src/number_theory/arithmetic_function.lean

```diff
@@ -623,16 +643,23 @@ begin
   simp [pow, (ne_of_lt (nat.succ_pos k)).symm],
 end
 
+lemma pow_zero_eq_zeta : pow 0 = ζ := by { ext n, simp }
+
 /-- `σ k n` is the sum of the `k`th powers of the divisors of `n` -/
 def sigma (k : ℕ) : arithmetic_function ℕ :=
 ⟨λ n, ∑ d in divisors n, d ^ k, by simp⟩
 
 localized "notation `σ` := nat.arithmetic_function.sigma" in arithmetic_function
 
-@[simp]
```

Comment:
I am removing this `simp` lemma as I think the right hand side is not necessarily simpler than the left - in some cases it's preferable to use abstract properties about sigma (eg multiplicativity) rather than use the sum formula.